platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/snapshot/DefaultStateSnapshotManager.java

```diff
@@ -210,6 +210,10 @@ private void checkSignatures(@NonNull final SignedState reservedState) {
 
         // don't log an error if this is a freeze state. they are expected to lack signatures
         if (reservedState.isFreezeState()) {
+            final double signingWeightPercent = ((double) reservedState.getSigningWeight())
+                    / reservedState.getAddressBook().getTotalWeight()
+                    * 100;
```

Comment:
Minor nits for all the computations in this fix:
1. Might use `100.`. I'm sure javac will auto-expand the integer to double at compile time, but having all values as `double` makes the intent clearer to human readers.
2. Might group the expressions as `((double) x / y) * 100.`. Again, the current expression w/o any parentheses makes sense given the operators precedence in Java, but the `(/)*` grouping is easier to understand for a human w/o having to remember or double-check the operators precedence.

Reply:
Just to make sure I am following you... this is what you are proposing:
```
final double signingWeightPercent = (((double) reservedState.getSigningWeight())
        / ((double) reservedState.getAddressBook().getTotalWeight()))
        * 100.0;
// and...
final double signingWeight1Percent = (((double) signingWeight1) / ((double) totalWeight1)) * 100.0;
```

Reply:
Overall, yes, that looks super clear. I'd even be okay with a slightly simpler version of literally `( (double) x / y) * 100.0` w/o extra parentheses around `(double) x` and w/o converting the `y` to `double`. But I still like your version where everything explicitly made a `double` and there's no doubts about the order of operations at all. +1!

Reply:
Done!